<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="bg">
      <property name="styleSheet">
       <string notr="true">#bg{
background-color:rgb(19, 19, 19);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="header">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
color: rgb(255,255,255);
background: black;
border:none;
}
#QPushButton{
color: rgb(255,255,255);
background: transparent;
border:none;
}
#QPushButton-hover{
background: white;
border:none;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="iconic.qrc">
                 <normaloff>:/icon/feather (1)/minus.svg</normaloff>:/icon/feather (1)/minus.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="iconic.qrc">
                 <normaloff>:/icon/feather (1)/maximize.svg</normaloff>:/icon/feather (1)/maximize.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="iconic.qrc">
                 <normaloff>:/icon/feather (1)/x.svg</normaloff>:/icon/feather (1)/x.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>45</width>
            <height>441</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(10,10,10);
color:rgb(255, 255, 255);
border:none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QToolButton" name="toolButton">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>5</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/menu.svg</normaloff>:/icon/feather (1)/menu.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_2">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>50</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/user.svg</normaloff>:/icon/feather (1)/user.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_3">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>100</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/file-plus.svg</normaloff>:/icon/feather (1)/file-plus.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_4">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>150</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/camera.svg</normaloff>:/icon/feather (1)/camera.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_5">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>200</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/eye.svg</normaloff>:/icon/feather (1)/eye.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_6">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>250</y>
             <width>27</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="iconic.qrc">
             <normaloff>:/icon/feather (1)/printer.svg</normaloff>:/icon/feather (1)/printer.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_5">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>0</y>
            <width>741</width>
            <height>541</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>190</y>
             <width>281</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>COOL AS EYES</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </widget>
         <zorder>frame_5</zorder>
         <zorder>frame_4</zorder>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="footer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="iconic.qrc"/>
 </resources>
 <connections/>
</ui>
